lib/utils/log/slog_handler.go

```diff
@@ -553,6 +556,32 @@ func NewSlogJSONHandler(w io.Writer, cfg SlogJSONHandlerConfig) *SlogJSONHandler
 	}
 }
 
+const (
+	traceID = "trace_id"
+	spanID  = "span_id"
+)
+
+func addTracingContextToRecord(ctx context.Context, r *slog.Record) {
+	span := oteltrace.SpanFromContext(ctx)
+	if span == nil || !span.IsRecording() {
+		return
+	}
+
+	spanContext := span.SpanContext()
+	if spanContext.HasTraceID() {
+		r.AddAttrs(slog.String(traceID, spanContext.TraceID().String()))
+	}
+
+	if spanContext.HasSpanID() {
+		r.AddAttrs(slog.String(spanID, spanContext.SpanID().String()))
+	}
```

Comment:
`trace.TraceID` and `trace.SpanID` implement `String` and `MarshalJSON`, both resulting in a hex string, so I think we can leave both as an `Any`. `SpanContext` also implements `MarshalJSON`, actually... 👀 

Do we know at this point if the record will be rendered by the `JSONHandler`, or is that something that's only decided later by the inner `Handle()`?

```suggestion
	if spanContext.HasTraceID() {
		r.AddAttrs(slog.Any(traceID, spanContext.TraceID()))
	}

	if spanContext.HasSpanID() {
		r.AddAttrs(slog.Any(spanID, spanContext.SpanID()))
	}
```